srcpkgs/lgi-LuaJIT/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'lgi-LuaJIT'
+pkgname=lgi-LuaJIT
+version=0.9.2
+revision=1
+wrksrc="lgi-${version}"
+build_style=gnu-makefile
+make_install_args="LUA_VERSION=5.1"
+hostmakedepends="pkg-config"
+makedepends="libglib-devel libffi-devel lua51-devel
+ gobject-introspection gir-freedesktop"
+depends="LuaJIT gir-freedesktop"
+short_desc="LuaJIT bindings for gnome/gobject using gobject-introspection library"
+maintainer="mustaqim <mustaqim@pm.me>"
+license="MIT"
+homepage="https://github.com/pavouk/lgi"
+distfiles="https://github.com/pavouk/lgi/archive/$version.tar.gz"
+checksum=cfc4105482b4730b3a40097c9d9e7e35c46df2fb255370bdeb2f45a886548c4f
+conflicts="lgi"
+
+post_install() {
+	vbin tools/dump-typelib.lua dump-typelib
```

Comment:
For this one, I inspected the original `lgi` and this package. The only file that is conflict is the `tools/dump-typelib.lua`. Maybe we should change this to `dump-tpyelib5.1`, the one in the original `lgi` to `dump-typelib5.3` and add and alternative group?